sandbox/src/main/java/org/robolectric/internal/bytecode/ClassInstrumentor.java

```diff
@@ -229,15 +229,19 @@ private boolean isJacocoInstrumented(MethodNode ctor) {
       if ((node instanceof LdcInsnNode && ((LdcInsnNode) node).cst instanceof ConstantDynamic)) {
         ConstantDynamic cst = (ConstantDynamic) ((LdcInsnNode) node).cst;
         return cst.getName().equals("$jacocoData");
-      } else if (insns.length > 1
-          && insns[0] instanceof LabelNode
-          && insns[1] instanceof MethodInsnNode) {
-        return "$jacocoInit".equals(((MethodInsnNode) insns[1]).name);
       }
     }
     return false;
   }
 
+  private boolean isJacocoInstrumentedInConstructorCallSuper(InsnList ctor) {
+    AbstractInsnNode[] insns = ctor.toArray();
+    if (insns.length > 1 && insns[0] instanceof LabelNode && insns[1] instanceof MethodInsnNode) {
```

Comment:
Sometimes in Jacoco-instrumented constructors, there is no LabelNode. insns[0] is a MethodInsnNode.

Reply:
Yeap, I saw that's optimized in master code. I simple merged and optimized the impl.